README.md

```diff
@@ -1,157 +1,15 @@
-
-Execute the `new` command and provide the name of your project as the only
-argument:
-
-```bash
-# Linux, Mac OS X
-$ symfony new my_project
-
-# Windows
-c:\> php symfony new my_project
-```
-
-**2. Start a new project with the latest Symfony LTS (Long Term Support) version**
-
-Execute the `new` command and provide the name of your project as the first
-argument and `lts` as the second argument. The installer will automatically
-select the most recent LTS (*Long Term Support*) version available:
-
-```bash
-# Linux, Mac OS X
-$ symfony new my_project lts
-
-# Windows
-c:\> php symfony new my_project lts
-```
-
-**3. Start a new project based on a specific Symfony branch**
-
-Execute the `new` command and provide the name of your project as the first
-argument and the branch number as the second argument. The installer will
-automatically select the most recent version available for the given branch:
-
-```bash
-# Linux, Mac OS X
-$ symfony new my_project 2.8
-
-# Windows
-c:\> php symfony new my_project 2.8
-```
-
-**4. Start a new project based on a specific Symfony version**
-
-Execute the `new` command and provide the name of your project as the first
-argument and the exact Symfony version as the second argument:
-
-```bash
-# Linux, Mac OS X
-$ symfony new my_project 2.8.1
-
-# Windows
-c:\> php symfony new my_project 2.8.1
-```
-
-**5. Install the Symfony demo application**
-
-The Symfony Demo is a reference application developed using the official Symfony
-Best Practices:
+**The Symfony Installer is no longer recommended to start new projects based on
+the Symfony full-stack framework.** Instead, use [Composer](https://getcomposer.org/):
 
 ```bash
-# Linux, Mac OS X
-$ symfony demo
-
-# Windows
-c:\> php symfony demo
-```
-
-Updating the installer
-----------------------
-
-New versions of the Symfony Installer are released regularly. To update your
-installer version, execute the following command:
-
-```bash
-# Linux, Mac OS X
-$ symfony self-update
-
-# Windows
-c:\> php symfony self-update
-```
-
-> **NOTE**
->
-> If your system requires the use of a proxy server to download contents, the
-> installer tries to guess the best proxy settings from the `HTTP_PROXY` and
-> `http_proxy` environment variables. Make sure any of them is set before
-> executing the Symfony Installer.
-
-Troubleshooting
----------------
-
-### SSL and certificates issues on Windows systems
-
-If you experience any error related with SSL or security certificates when using
-the Symfony Installer on Windows systems:
-
-1) Check that the OpenSSL extension is enabled in your `php.ini` configuration:
-
-```ini
-; make sure that the following line is uncommented
-extension=php_openssl.dll
-```
-
-2) Check that the path to the file that contains the security certificates
-exists and is defined in `php.ini`:
+# Symfony 2.x and 3.x
+$ composer create-project symfony/framework-standard-edition my_project_name
```

Comment:
Wouldn't  be good to add information on how to create a project with a certain version like
`composer create-project symfony/framework-standard-edition my_project_name 2.8`
or
`composer create-project symfony/framework-standard-edition my_project_name 3.4`